suggestion/package.json

```diff
@@ -23,7 +23,7 @@
         "request": "^2.88.0"
     },
     "devDependencies": {
-        "@types/jest": "^24.0.23",
+        "@types/jest": "^24.9.1",
```

Comment:
Was version change on purpose?

Reply:
Yeah, it was an accident. Changed to 24.0.23.